add node for translating between NavSatFix and GPSFix

volkandre opened the pull request:
I added a node, which can convert to and from NavSatFix. Maybe this can be useful and helps to increase the acceptance of GPSFix messages. The launch file explains the usage and gives an example on the parametrization.

pjreed commented:
Looks good to me; could you also make the CMakeLists.txt in gps_common install it?

volkandre commented:
I added some lines to CMakeLists.txt and package.xml and moved the script to nodes/ instead of scripts/. This should be more conform to the style guide now. I also added a usage example to Readme.md.
